protopipe/scripts/make_performance_EventDisplay.py

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/env python
+
+import logging
+import os
+import astropy.units as u
+from astropy import table
+from astropy.io import fits
+import argparse
+import pandas as pd
+import numpy as np
+import operator
+
+from protopipe.pipeline.utils import load_config
+from protopipe.perf.utils import read_DL2_pyirf
+
+from pyirf.binning import (
+    add_overflow_bins,
+    create_bins_per_decade,
+    create_histogram_table,
+)
+from pyirf.spectral import (
+    calculate_event_weights,
+    PowerLaw,
+    CRAB_HEGRA,
+    IRFDOC_PROTON_SPECTRUM,
+    IRFDOC_ELECTRON_SPECTRUM,
+)
+
+from pyirf.sensitivity import calculate_sensitivity, estimate_background 
+
+from pyirf.utils import calculate_theta, calculate_source_fov_offset
+from pyirf.cuts import calculate_percentile_cut, evaluate_binned_cut
+from pyirf.cut_optimization import optimize_gh_cut
+from pyirf.irf import (
+    effective_area_per_energy,
+    energy_dispersion,
+    psf_table,
+    background_2d,
+)
+from pyirf.io import (
+    create_aeff2d_hdu,
+    create_psf_table_hdu,
+    create_energy_dispersion_hdu,
+    create_rad_max_hdu,
+    create_background_2d_hdu,
+)
+from pyirf.benchmarks import energy_bias_resolution, angular_resolution
+
+log = logging.getLogger("pyirf")
+
+def main():
+    
+    # Read arguments
+    parser = argparse.ArgumentParser(description='Make performance files')
+    parser.add_argument('--config_file', type=str, required=True, help='')
+    parser.add_argument(
+        '--obs_time',
+        type=str,
+        required=True,
+        help='Observation time, should be given as a string, value and astropy unit separated by an empty space'
+    )
+    mode_group = parser.add_mutually_exclusive_group()
+    mode_group.add_argument('--wave', dest="mode", action='store_const',
+                            const="wave", default="tail",
+                            help="if set, use wavelet cleaning")
+    mode_group.add_argument('--tail', dest="mode", action='store_const',
+                            const="tail",
+                            help="if set, use tail cleaning, otherwise wavelets")
+    args = parser.parse_args()
+
+    # Read configuration file
+    cfg = load_config(args.config_file)
+
+    # Add obs. time in configuration file
+    str_obs_time = args.obs_time.split()
+    cfg['analysis']['obs_time'] = {'value': float(str_obs_time[0]), 'unit': str(str_obs_time[-1])}
+
+    # Create output directory if necessary
+    outdir = os.path.join(cfg['general']['outdir'], 'irf_{}_Time{:.2f}{}'.format(
+        args.mode,
+        cfg['analysis']['obs_time']['value'],
+        cfg['analysis']['obs_time']['unit'])
+    )
+    if not os.path.exists(outdir):
+        os.makedirs(outdir)
+
+    indir = cfg['general']['indir']
+    template_input_file = cfg['general']['template_input_file']
+    
+    # Informations from config file
+    site = cfg['General']['site']
+    config_array = cfg['General']['array']
```

Comment:
also here, it is true that it can be read from `analysis.yaml`, but that variable is used there to select the subarrays to input in `ctapipe.io.EventSource`.
Since here you use them only to give a proper output name, I would migrate/copy such variables in `performance.yaml`, where you can also specify other things such as `baseline` (which is important to specify)